protos/backend_service.proto

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+syntax = "proto3";
+
+package durabletask.protos.backend.v1;
+
+option csharp_namespace = "Microsoft.DurableTask.Protobuf";
+option java_package = "com.microsoft.durabletask.implementation.protobuf";
+option go_package = "/internal/protos";
+
+import "orchestrator_service.proto";
+
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/empty.proto";
+
+// gRPC service used by Durable Task Framework (DTFx) backend implementations.
+// The RPCs in this service are used by DTFx clients to manage orchestrations.
+service ClientBackendService {
+    // Creates a new orchestration instance.
+    rpc CreateOrchestrationInstance (CreateInstanceRequest) returns (CreateInstanceResponse);
+
+    // Sends an event to an orchestration instance. This RPC is used for raising external events to orchestrations
+    // and for sending orchestration lifecycle events, such as terminate, suspend, resume, etc.
+    rpc AddOrchestrationEvent (AddOrchestrationEventRequest) returns (google.protobuf.Empty);
```

Comment:
Strongly recommend avoiding `google.protobuf.Empty`, instead create an empty message `AddOrchestrationEventResponse`

Reply:
I understand that either option makes no difference on the wire protocol, but only when recompiling the source code that uses it. However, if we decide we need to use a message that contains fields, we likely already need to make code changes anyways to consume that field. I therefore don't see much benefit to creating empty messages, but mostly downsides since it clutters the protobuf definitions.

Reply:
Actually, I've changed my mind about this. My reasoning doesn't take into consideration the fact that the compiled types are shipped as part of an independent library, in which case switching from `Empty` to something else is a breaking change for the library, thus requiring a major version update. I'd like to avoid major version updates for libraries whenever possible, so let's go with dedicated types as you originally suggested.